src/main/thrift/parquet.thrift

```diff
@@ -626,11 +628,9 @@ struct FileMetaData {
   6: optional string created_by
 
   /**
-   * Sort order used for each column in this file.
-   *
-   * If this list is not present, then the order for each column is assumed to
```

Comment:
And which fields does Impala read the statistics from if the column order is not set, `min` and `max` or `min-value` and `max-value`?

The fact that the statistics are in the `min_value` and `max_value` fields already shows that they are the new kind of statistics and were not written by older versions of parquet-mr.